src/cmd/services/m3coordinator/ingest/write.go

```diff
@@ -449,21 +462,26 @@ func (d *downsamplerAndWriter) WriteBatch(
 			}
 
 			d.metrics.written.report(value.Attributes.Source)
+			if len(storagePolicies) > 1 {
+				d.metrics.multiPolicyWritten.reportInc(value.Attributes.Source, len(storagePolicies))
```

Comment:
data shows this isn't happening: https://m3-aws-us-west-2.dev.databricks.com/pantheon?g0.expr=sum%28rate%28coordinator_downsampler_metrics_multi_policy_written%5B5m%5D%29%29&g0.tab=0&g0.Humanize+Labels=false&g0.Stacked=false&g0.UTC=true&g0.Full+Width=false&g0.startStr=now-1h&g0.endStr=now&g0.range_input=1h

Reply:
I saw that. I'd prefer to keep this metric around in case something changes later.

Reply:
got it, is this a dup with https://github.com/databricks/m3/pull/80/files#r1588002935